deepdrivemd/utils.py

```diff
@@ -89,3 +90,50 @@ def bestk(
     # Only sorts an array of size k
     sort_inds = np.argsort(best_values)
     return best_values[sort_inds], best_inds[sort_inds]
+
+
+def t2Dto1D(A):
```

Comment:
Please correct me if I'm wrong, but I believe these functions are for processing the contact maps. If so, they should be encapsulated with the contact map logic. deepdrivemd/utils.py is more for common code that can be used throughout deepdrivemd or generic helper functions. Contact maps are only one form of model inputs and are not computed for the AAE model, or other models in general.

Reply:
Those functions are used in several logical components to pack and unpack contact maps. I think, they belong either in utils or in a separate module under misc.